src/GCHP_GridComp/GCHPctmEnv_GridComp/GCHPctmEnv_GridCompMod.F90

```diff
@@ -589,11 +606,30 @@ subroutine Run ( GC, IMPORT, EXPORT, CLOCK, RC )
       _VERIFY(STATUS)
       call MAPL_GetPointer ( IMPORT,      VA,    'VA',  RC=STATUS )
       _VERIFY(STATUS)
-      call MAPL_GetPointer ( IMPORT,   SPHU0,  'SPHU1', RC=STATUS )
+      call MAPL_GetPointer ( IMPORT,   SPHU0_IMPORT,  'SPHU1', RC=STATUS )
       _VERIFY(STATUS)
-      call MAPL_GetPointer ( IMPORT,   SPHU1,  'SPHU2', RC=STATUS )
+      call MAPL_GetPointer ( IMPORT,   SPHU1_IMPORT,  'SPHU2', RC=STATUS )
       _VERIFY(STATUS)
 
+      ! Get local dimensions
+      is = lbound(UA,1); ie = ubound(UA,1)
+      js = lbound(UA,2); je = ubound(UA,2)
+      lm = size  (UA,3)
+
+      ALLOCATE(SPHU0(is:ie,js:je,lm), STAT=STATUS); 
```

Comment:
I may be missing something, but I'm not seeing why it is necessary to allocate the local arrays to store the vertically flipped imports. Aren't the humidities only used here: https://github.com/geoschem/GCHP/blob/efbba117948f4b868925e4f8f90d2e5cda3f1957/src/GCHP_GridComp/GCHPctmEnv_GridComp/GCHPctmEnv_GridCompMod.F90#L677-L697? If yes, can't the L index be swapped to with the vertically flipped equivalent?